open-api/rest-catalog-open-api.yaml

```diff
@@ -3324,6 +3348,211 @@ components:
           type: integer
           format: int64
 
+    BooleanTypeValue:
+      type: boolean
+      example: true
+
+    IntegerTypeValue:
+      type: integer
+      example: 42
+
+    LongTypeValue:
+      type: integer
+      format: int64
+      example: 9223372036854775807
+
+    FloatTypeValue:
+      type: number
+      format: float
+      example: 3.14
+
+    DoubleTypeValue:
+      type: number
+      format: double
+      example: 123.456
+
+    DecimalTypeValue:
+      type: string
+      description:
+        "Decimal types are serialized as strings using BigDecimal to maintain precision. For scales greater than 
```

Comment:
I think we should avoid saying Java specific things in the description. Maybe something like:

> Decimal types are serialized as strings. Decimals with a positive scale is serialized as numeric plain text, and decimals with a negative scale are serialized in exponential notation.

Reply:
also a general comment about the type value descriptions, it is not a "type" that is serialized to something. It is a "type value" that is. So we should say "Decimal type values are serialized as ..." instead of "Decimal types are serialized as ...". Please check the other descriptions to make sure it is consistent.